### ISSUE TITLE
what the set is the pretrained model trained on?

### ISSUE THREAD

**Issue by LeopoldACC** [edwardzhou130/PolarSeg#34]
As the most paper's experiment setting about kitti is that use sequence 00-10 as train set except 08 as validation set?does the pretrained model trained on `the whole 00-10 sequence` or `sequence 00-10 as train set except 08 as validation set`?
Thks!

**Comment by edwardzhou130** [comment]
This is the split of the sequences:
https://github.com/edwardzhou130/PolarSeg/blob/d7088e3965e845fb8a563ea759b6c447b65d2506/semantic-kitti.yaml#L186-L211

**Comment by LeopoldACC** [comment]
that means sequence `08` not used to update the weight of the network,Yeah?
Thks!

**Comment by edwardzhou130** [comment]
``08`` will not be used to update the model. It is used to evaluate the performance of the model and decide whether the model should be saved.

**Comment by LeopoldACC** [comment]
Thks!
